lib/engine/game/g_1880/game.rb

```diff
@@ -712,7 +713,7 @@ def lessee
         def round_description(name, round_number = nil)
           round_number ||= @round.round_num
           description = super
-          description += ".#{round_number}" if @round.is_a?(Engine::Round::Operating)
+          description += ".#{round_number}" if name == self.class::OPERATING_ROUND_NAME
           description += " - Train Marker at #{@train_marker.name}" if @train_marker && !@end_game_triggered
```

Comment:
needed change, timing issue.
When round_description is called for logging during an SR, it's still the OR, so the previous conditional will resolve to true even when i want to log the SR round. 
https://github.com/tobymao/18xx/blob/master/lib/engine/game/base.rb#L2627